urdfenvs/urdf_common/urdf_env.py

```diff
@@ -534,11 +541,13 @@ def add_sensor(self, sensor: Sensor, robot_ids: List) -> None:
 
     def reset(
         self,
+        seed = None, 
+        options = None,
```

Comment:
Can you add the type annotation for those?

Reply:
Done.